package.json

```diff
@@ -2,18 +2,20 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "dependencies": {
-    "expo": "^25.0.0",
+    "axios": "^0.17.1",
+    "expo": "24.0.0",
     "firebase": "^4.9.0",
     "ionicons": "^3.0.0",
-    "react": "16.2.0",
-    "react-native": "https://github.com/expo/react-native/archive/sdk-25.0.0.tar.gz",
+    "react": "16.0.0",
+    "react-native": "https://github.com/expo/react-native/archive/sdk-24.0.0.tar.gz",
     "react-native-elements": "^0.19.0",
     "react-native-linear-gradient": "^2.4.0",
     "react-native-vector-icons": "^4.5.0",
-    "react-navigation": "^1.0.0-beta.27",
+    "react-navigation": "1.0.0-beta.21",
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
+    "redux-persist": "^4.10.2",
```

Comment:
Above are the other changes that were needed to downgrade expo. I also added `axios` for asyn operations (native `fetch` can also do the job, but `axios` has some nice helper functionality), and `redux-persist` for the next big step which is offline data persistence.